spring-cloud-starter-stream-processor-twitter-sentiment/src/test/java/org/springframework/cloud/stream/app/twitter/sentiment/processor/twitter/TwitterSentimentTensorflowProcessorIntegrationTests.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.twitter.sentiment.processor.twitter;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.stream.app.twitter.sentiment.processor.TwitterSentimentProcessorConfiguration;
+import org.springframework.cloud.stream.messaging.Processor;
+import org.springframework.cloud.stream.test.binder.MessageCollector;
+import org.springframework.context.annotation.Import;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringRunner;
+
+/**
+ * Integration Tests for TwitterSentimentTensorflowProcessor
+ *
+ * @author Christian Tzolov
+ */
+@SuppressWarnings("SpringJavaAutowiringInspection")
+@RunWith(SpringRunner.class)
+@SpringBootTest(
+		webEnvironment = SpringBootTest.WebEnvironment.NONE,
+		properties = {
+				"tensorflow.modelLocation=http://dl.bintray.com/big-data/generic/minimal_graph.proto",
```

Comment:
I acknowledge that this is a problem. The reason they are not in the project is the size (~100MB).  
If we add the trained model to the source code then the benefit is that we can have a default pre-trained model OOTB. So the twitter-sentiment can be deployed and used as it is (for EN at least).  
One possible alternative is to to create a special `resource` pom artefacts released and resolved from our maven repo? 
Another option concerning testing is to try to create an empty (not trained) model for test purposes. It should be much smaller. 
WDYT?